### PR TITLE
Fixed typescript error from resolving unspecified in exports map .ts file

### PR DESCRIPTION
**Pull request by XantreGodlike**
Should fix: #450

### CODE REVIEW

**Review by rschristian — APPROVED** [review]
Cheers, let me know if you're good w/ having this merged as it's still set to be a draft

**Comment by XantreGodlike** [comment]
Since @paul-sachs confirmed this is working - undrafted

File: .changeset/wise-toes-joke.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals-react": patch
+---
+
+Fixed typescript error from resolving unspecified in exports map .ts file
```
Comment (rschristian) [inline .changeset/wise-toes-joke.md:5]:
```suggestion
Ensure types are resolved against built `.d.ts` rather than source `.ts`
```

**Comment by rschristian** [comment]
Sorry for the close & reopen, the CI seemingly got stuck

**Pull request merged**